docs/build-hibench.md

```diff
@@ -28,7 +28,7 @@ Because some Maven plugins cannot support Scala version perfectly, there are som
 
 
 ### Specify Spark Version ###
-To specify the spark version, use -Dspark=xxx(1.6, 2.0 or 2.1). By default, it builds for spark 2.0
+To specify the spark version, use -Dspark=xxx(1.6, 2.0, 2.1 or 2.2). By default, it builds for spark 2.0
```

Comment:
Why don't we use Spark 2.2 as default?

Reply:
We may not want to update this to the latest version and user is always encouraged to specify the spark version when building HiBench. We can update this when we release the next HiBench version.